docs/configuration.md

```diff
@@ -777,24 +758,22 @@ object they're wrapping. This means a registry middleware must implement the
 Currently only one middleware, `cloudfront`, a storage middleware, is supported
 in the registry implementation.
 
-```yaml
-middleware:
-	registry:
-		- name: ARegistryMiddleware
-		  options:
-			foo: bar
-	repository:
-		- name: ARepositoryMiddleware
-		  options:
-			foo: bar
-	storage:
-		- name: cloudfront
-		  options:
-			baseurl: https://my.cloudfronted.domain.com/
-			privatekey: /path/to/pem
-			keypairid: cloudfrontkeypairid
-			duration: 3000
-```
+    middleware:
+      registry:
+        - name: ARegistryMiddleware
+          options:
+          foo: bar
```

Comment:
I think you need to check the indentation in this example; for example,  "foo" was a sub of "options" before your change?